Firestore/Example/Firestore_Benchmarks_iOS/FSTLevelDBBenchmarkTests.mm

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import "../../../../../../../Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/Headers/XCTest.h"
+
+#include "benchmark/benchmark.h"
+
+#import "Firestore/Source/Core/FSTTypes.h"
+#import "Firestore/Source/Local/FSTLevelDB.h"
+#import "Firestore/Source/Local/FSTLevelDBKey.h"
+#import "Firestore/Source/Local/FSTLocalSerializer.h"
+#import "Firestore/Source/Model/FSTDocumentKey.h"
+#import "Firestore/Source/Remote/FSTSerializerBeta.h"
+#include "Firestore/core/src/firebase/firestore/local/leveldb_transaction.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+using firebase::firestore::local::LevelDbTransaction;
+using firebase::firestore::model::DatabaseId;
+
+// Pre-existing document size
+static const int kDocumentSize = 1024 * 2;  // 2 kb
```

Comment:
Optional nit: instead of using `static` variables and functions, consider wrapping them in an anonymous namespace. It achieves the same thing, but is slightly less verbose, and has the advantage of working being more universal (`static` cannot be applied to `class`/`struct`).

Reply:
Done.